providers/openshift/provider.go

```diff
@@ -29,6 +30,13 @@ import (
 	authorizationv1beta1 "k8s.io/client-go/pkg/apis/authorization/v1beta1"
 )
 
+// httpClientCache stores httpClient objects so that new client does not have to
+// be created on each request just to prevent CAs content changed
+// key is bytes of the hashed metadata of the files for CAs the client was
+// created with
+// NOTE: the entries of the map are currently not being cleaned up
+var httpClientCache sync.Map
```

Comment:
can't this be a field in OpenShiftProvider? globals are evil.

Reply:
fixed